packages/fxa-content-server/tests/server/l10n-entrained.js

```diff
@@ -4,14 +4,14 @@
 const { registerSuite } = intern.getInterface('object');
 const assert = intern.getPlugin('chai').assert;
 const routesHelpers = require('./helpers/routesHelpers');
-const fxaShared = require('fxa-shared');
+const languages = require('../../../../libs/shared/l10n/src/lib/supported-languages.json');
 const dnshook = require('./lib/dnshook');
 
 var checkHeaders = routesHelpers.checkHeaders;
 var extractAndCheckUrls = routesHelpers.extractAndCheckUrls;
 var makeRequest = routesHelpers.makeRequest;
 
-var languages = fxaShared.l10n.supportedLanguages;
+//var languages = fxaSharedL10n.supportedLanguages;
```

Comment:
✂️

Reply:
Agreed, let's delete instead of commenting out.